src/resources/extensions/quarto/confluence/publish.lua

```diff
@@ -35,8 +35,13 @@ function Writer (doc, opts)
       return pandoc.RawInline('html', renderString)
     end,
     Link = function (link)
+      local renderedLinkContent =
+        pandoc.write(pandoc.Pandoc(link.content), "html")
+
```

Comment:
Thanks for the tip @dragonstyle.  For some reason link contents are having `\n` line breaks injected into the results.  I didn't seen any pandoc write options around this so I'm a bit confused as to where they are coming from.